react-forms/my-app/src/RegistrationFormUncontrolled.tsx

```diff
@@ -0,0 +1,28 @@
+import { FormEvent } from 'react';
+
+export function RegistrationFormUncontrolled() {
+  const handleSubmit = (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    const form = event?.currentTarget;
```

Comment:
The `?` here is not needed since we know that `event` will never be `undefined`.